modules/smartclipBidAdapter.js

```diff
@@ -0,0 +1,359 @@
+import * as utils from '../src/utils.js';
+import { Renderer } from '../src/Renderer.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import { VIDEO } from '../src/mediaTypes.js';
+const BIDDER_CODE = 'smartx';
+const URL = '//bid.sxp.smartclip.net/bid/1000';
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['smartx'],
```

Comment:
If the biddercode and the alias are the same, the alias isn't needed.

Reply:
Done